### PR TITLE
General Enhancements to stable version documentation

### PR DESCRIPTION
**Pull request by fantandrea**
Fix 8 out 10 issues collected in Epic #112 to update current stable documentation.

Changes need to be pushed also on current development branch.

As per comment in Linkurious/linkurious-frontend#2662, the link of the Public API Version has been updated, and the sample result in the doc as well.

### CODE REVIEW

File: content/admin/content/access/sso-azure.md [hunk 1]
```diff
@@ -6,13 +6,14 @@ as an external authentication provider.
 To set up Linkurious authentication with Microsoft Azure Active Directory, follow these steps:
 
 1. Create a new app called `Linkurious` in Azure Active Directory on [Azure Portal](https://portal.azure.com)
-2. From the Azure Portal, obtain the following parameters:
+2. Assign the `Directory.Read.All` access right to the new app (**notice:** an azure admin's approval is needed)
```
Comment (davidrapin) [inline content/admin/content/access/sso-azure.md]: capitalize "Azure" everywhere, it's a brand.

File: content/admin/content/getting-started/download.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-Log in with the username and password created during the purchase process.
-You will then see the downloadable files:
- - For Windows: `linkurious-windows-v{{package.version}}.zip`
- - For Linux: `linkurious-linux-v{{package.version}}.zip`
- - For Mac OS X: `linkurious-osx-v{{package.version}}.zip`
+Log in with the username and password created during the purchase process and then go
+to the download section for the specific license (in case of multiple one), it will be
+possible to download the package for the correct platform.
+
+> Only specific roles will have access to the download section, in particular technical people
```
Comment (davidrapin) [inline content/admin/content/getting-started/download.md]: this is incorrect, both BO and TC have access to the download section, this warning can be removed

File: content/admin/content/troubleshoot/support.md [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
 
 ![](download-report.png)
+
+# Manual process to collect logs
+
+It can happend that the system fails to start due an error. Since the impossibility to download the automatically generated *report*, the following files should be added manually to a compressed archive:
+* **The application Logs:** the `manager` folder stored in `<linkurious-installation>/data`
+* **The application Configuration (optional):** the `production.json` file stored in `<linkurious-installation>/data/config`
+
+  > The `production.json` file may contain sensitive data like passwords which can be stored as plain text or encrypted based on the system configuration.
```
Comment (davidrapin) [inline content/admin/content/troubleshoot/support.md]: grammar:

> Depending on your situation, the configuration file (`production.json`) may sometimes contain clear-text passwords. Consider redacting such password before sharing your configuration with the support team.

File: content/admin/content/troubleshoot/support.md [hunk 4]
```diff
@@ -1,10 +1,25 @@
...
 
 At the end of the page, click *Download Report*:
 
 ![](download-report.png)
+
+# Manual process to collect logs
+
+It can happend that the system fails to start due an error. Since the impossibility to download the automatically generated *report*, the following files should be added manually to a compressed archive:
+* **The application Logs:** the `manager` folder stored in `<linkurious-installation>/data`
+* **The application Configuration (optional):** the `production.json` file stored in `<linkurious-installation>/data/config`
```
Comment (davidrapin) [inline content/admin/content/troubleshoot/support.md]: why capitalize "configuration"?

File: content/shared/assets/drop-menu.js [hunk 5]
```diff
@@ -34,15 +34,15 @@
     loadAvailableVersions(rootPath, function(err, versions) {
       if (err) {
         console.log('XHR error: ' + (err.message ? err.message : JSON.stringify(err)));
-        versions = ['latest'];
+        versions = ['latest stable'];
```
Comment (davidrapin) [inline content/shared/assets/drop-menu.js]: the "last stable" folder does not exist, revert this change

File: content/shared/assets/drop-menu.js [hunk 6]
```diff
@@ -34,15 +34,15 @@
...
       } else {
-        versions.unshift('latest');
+        versions.push('latest stable');
```
Comment (davidrapin) [inline content/shared/assets/drop-menu.js]: same here, the folder is "latest"

File: content/shared/assets/drop-menu.js (hunk 5 above)
Comment (davidrapin) [reply content/shared/assets/drop-menu.js]: I you want to use the "latest" folder but display it a "latest folder", you need to change this code a bit more to distinguish between the displayed name in the list and the target folder name.

File: content/user/content/design/index.md [hunk 7]
```diff
@@ -13,7 +13,9 @@ The Design panel will open with the **Styles** and **Nodes** tabs pre-selected:
 
 > Styles set in a new visualization are automatically
 re-applied to newly created visualizations.
-Users can reset the styles at any time.
+Users can reset the styles at any time from the **gear** icon at the bottom-rigth part of the **Styles** or **Captions** panel.
```
Comment (davidrapin) [inline content/user/content/design/index.md]: Users can reset the styles at any time ... => Users can reset their styles default styles at any time ...

File: content/admin/content/troubleshoot/support.md (hunk 4 above)
Comment (fantandrea) [reply content/admin/content/troubleshoot/support.md]: Was to emphasize the difference between `Logs` and `Configuration` if remove the capital, I should remove also from `Logs`, I can maybe set italic both words or leave as standard text if you don't think is useful to mark them

File: content/shared/assets/drop-menu.js (hunk 5 above)
Comment (fantandrea) [reply content/shared/assets/drop-menu.js]: The idea is indeed to just rename the text of link. When I tried locally I guess I was able to follow the link, I didn't notice I broke the link. I'll re-check.

File: content/admin/content/getting-started/download.md (hunk 2 above)
Comment (fantandrea) [reply content/admin/content/getting-started/download.md]: I noticed the same when writing, indeed I tried to avoid putting the real role names and to say who for sure has access rather than listing who has access.

The idea is to specify that if you don't see the download section is because you are in the wrong group. The other information is that (as per standard process) **at least** the technical people has this right.

If this is not clear and you think this is the correct place, maybe we can add the full role list and their grants.

File: content/admin/content/access/sso-azure.md (hunk 1 above)
Comment (fantandrea) [reply content/admin/content/access/sso-azure.md]: Fixed with 91339a5897ccba094ecd635a4421e12f0ca01813

File: content/admin/content/troubleshoot/support.md (hunk 3 above)
Comment (fantandrea) [reply content/admin/content/troubleshoot/support.md]: Fixed with 91339a5897ccba094ecd635a4421e12f0ca01813

File: content/admin/content/troubleshoot/support.md (hunk 4 above)
Comment (fantandrea) [reply content/admin/content/troubleshoot/support.md]: Fixed with 91339a5897ccba094ecd635a4421e12f0ca01813

File: content/user/content/design/index.md (hunk 7 above)
Comment (fantandrea) [reply content/user/content/design/index.md]: Fixed with 91339a5897ccba094ecd635a4421e12f0ca01813

File: content/shared/assets/drop-menu.js (hunk 5 above)
Comment (fantandrea) [reply content/shared/assets/drop-menu.js]: Fixed with 5d0575445710a8771fbd9e464df93100045f5682

File: content/shared/assets/drop-menu.js (hunk 6 above)
Comment (fantandrea) [reply content/shared/assets/drop-menu.js]: Fixed with 5d0575445710a8771fbd9e464df93100045f5682

File: content/admin/content/getting-started/download.md (hunk 2 above)
Comment (fantandrea) [reply content/admin/content/getting-started/download.md]: @davidrapin any feedback on how to proceed on this?

Comment (davidrapin) [reply content/admin/content/getting-started/download.md]: To clarify: **all** possible users/roles have access to the download section.

Comment (fantandrea) [reply content/admin/content/getting-started/download.md]: Ok got now. In this case was incorrect, I though there were at least a group with no download permission.

Fixed with 401b5a1a80d96b975c9e3d48599a5c3b9096c895

File: content/shared/assets/drop-menu.js (hunk 5 above)
Comment (davidrapin) [reply content/shared/assets/drop-menu.js]: added a comment to clarify why we have to do that

**Comment by fantandrea** [comment]
@davidrapin what is the standard process to delete the branch and close the issues?
1. Should I do it?
2. Is the merge of the branch a good trigger or should I wait something else (e.g. the commits in production)?

**Comment by davidrapin** [comment]
I usually delete the branch as soon as I merge the PR (I just did).
If nothing remains to be changed, you can close the issues (usually, the person who opened the issues validates/closes them when the changes have been made).

**Comment by fantandrea** [comment]
@davidrapin I was expecting to see this changes in production also for the current stable version 2.7.7, did I have false expectations? They are currently online for 2.8.2, any plan to rebuild the current version or will be added just from 2.7.8 ahead?

**Comment by davidrapin** [comment]
@fantandrea The changes will be visible from 2.7.8 and ahead (we currently do not re-generate the doc older versions). 
What is surprising is that 2.8.2 is already up-to-date, did you back-merge preprod-stable into preprod too? 

@francescoinfante did you merge preprod-stable into preprod at some point?

**Comment by fantandrea** [comment]
@davidrapin yes, I checked with francesco and I did it before the release of 2.8.2 f2dab56fb93ef91bc6a34397c2680fa789e10b9a (preprod-stable -> preprod) & e2597752dfbeb30ca296b9224fea2caab6a50db5 (preprod -> preprod-next)
